README.md

```diff
@@ -80,31 +80,84 @@ scripts/cmdeploy bench
 
 ## Overview of this repository
 
-This repository drives the development of chatmail services, 
-comprised of minimal setups of
-
-- [postfix smtp server](https://www.postfix.org)
-- [dovecot imap server](https://www.dovecot.org)
-
-as well as custom services that are integrated with these two:
-
-- `chatmaild/src/chatmaild/doveauth.py` implements
+This repository has four directories:
+
+- [cmdeploy](https://github.com/deltachat/chatmail/tree/main/cmdeploy)
+  is a collection of configuration files
+  and a [pyinfra](https://pyinfra.com)-based deployment script.
+- [chatmaild](https://github.com/deltachat/chatmail/tree/main/chatmaild)
```

Comment:
could you use a blank line to separate the list items? It doesn't make a difference in rendering but when reading the text source.